lib/core/src/lib/form/components/form-field/form-field.component.ts

```diff
@@ -88,14 +92,29 @@ export class FormFieldComponent implements OnInit, OnDestroy {
                     instance.fieldChanged.subscribe((field) => {
                         if (field && this.field.form) {
                             this.visibilityService.refreshVisibility(field.form);
-                            field.form.onFormFieldChanged(field);
+                            this.triggerFormFieldChanged(field);
```

Comment:
is it worth to have a method just for a single line of code?

Reply:
this method now is used in two places, so  thats why i move it to separate method, but i can agree that maybe its not worth